src/main/java/thePackmaster/powers/dimensiongate3pack/OuroborosPower.java

```diff
@@ -0,0 +1,63 @@
+package thePackmaster.powers.dimensiongate3pack;
+
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class OuroborosPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = makeID("OuroborosPower");
+    public static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    public static final String DESCRIPTIONS[] = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    private boolean triggeredThisTurn = false;
+
+    public OuroborosPower(AbstractCreature owner, int amount) {
+        super(POWER_ID, NAME, PowerType.BUFF, true, owner, amount);
+
+    }
+
+    @Override
+    public int onAttacked(DamageInfo info, int damageAmount) {
```

Comment:
Have you tested this working correctly?

Reply:
Yeah, the power works as described.  As long as you full-block any one source of damage during the turn you get a Strength.  

...but you wouldn't have written this without suspicion.  Is there an edge case I'm not considering?

Reply:
ah, i'm guessing the lack of a check on "was triggered this turn".  which I'll still add cause the power will flash on every full blocked source instead of just one, but it didn't affect functionality.